slackbridge/utils.py

```diff
@@ -36,6 +74,54 @@ def strip_nick(nick):
     ])
 
 
+def format_irc_message(text, users, channels):
```

Comment:
very well done.

Reply:
Thanks! It's mostly adapted from the [slack-irc project](https://github.com/ekmartin/slack-irc/blob/2b5ceb7ca7beb/lib/bot.js#L154) that has to accomplish a similar task.